backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java

```diff
@@ -75,7 +66,7 @@ public boolean IsImplemented() {
       String resourceId,
       String resourceOwnerId,
       Metadata requestHeaders) {
-    try (AuthServiceChannel authServiceChannel = new AuthServiceChannel()) {
+    try (AuthServiceChannel authServiceChannel = uac.getBlockingAuthServiceChannel()) {
```

Comment:
I'm pretty sure that, if you do this here, we'll attempt to do `authServiceChannel.close()` at the end of the `try`, which will close the shared object, right? Could you confirm?

Reply:
Yes, it will do automatically because `AuthServiceChannel` class has implemented AutoClose interface which is give one method `close()` where i am doing `authServiceChannel.close()` already.